Header header

float32 altitude
coax_client/BlobSequencePose[]  sequence_poses
